notebooks/DrizzlePac/Initialization/Initialization.ipynb

```diff
@@ -0,0 +1,319 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.environ['iref'] = './reference_files/references/hst/wfc3/'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now the DrizzlePac software will be able to locate the reference files it requires in the case that the user needs to update the geometric distortion information in the image header. These files are the IDCTAB (Instrument Distortion Correction Table, `*idc.fits`), the D2IMFILE (Column Correction Reference File, `*d2i.fits`), and the NPOLFILE (Non-polynomial Offsets Reference File, `*npl.fits`).  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[Top of Page](#title_ID)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Update World Coordinate System\n",
+    "\n",
+    "Before combining images with `drizzlepac.astrodrizzle` to make mosaics, the data may first need to be processed with `stwcs.updatewcs`. This task is required for populating older (pre-AstroDrizzle Archive pipeline data) WFC3, ACS, STIS, and WFPC2 images with linear and polynomial distortion correction information in a format compatible with AstroDrizzle. It is also required for data that was downloaded from MAST at some prior date and the user wishes to update the image headers to use a more recent (or custom) set of distortion reference files. \n",
+    "\n",
+    "The `updatewcs` task will update the header keywords with new WCS information and apply several distortion corrections from reference files. In general, data recently retrieved from MAST will have already had this step performed and does **not** need to be run. \n",
+    "\n",
+    "**WARNING: If you re-run an instrument calibration pipeline on raw data, or your files were retrieved from MAST long ago, you must run `updatewcs` or you will encounter errors.**\n",
+    "\n",
+    "As an example, if it were necessary to run `updatewcs` on our WFC3/UVIS file, we can update the WCS as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "stwcs.updatewcs.updatewcs('ib2j02n5q_flc.fits', use_db=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that we have set `use_db=False` in our call to `updatewcs`. The Astrometry Working Group at STScI has created infrastructure in `updatewcs` to include multiple astrometric solutions as additional extensions to HST FITS files. The default behavior is `use_db=True`. The Astrometric Database is a work in progress, and astrometric solutions are not yet available for all parts of the sky. In such cases, warnings will appear while using the Astrometric Database with `updatewcs`, but your data are still properly prepared for `astrodrizzle`."
```

Comment:
I find this wording confusing.  It's only at the end that it's stated what the "Astrometric Database" is, and therefore what the keyword is doing.  I suggest sort of "inverting the paragraph, or at least linking to a web site if there is one that describes the astrometric database and/or WG in more detail.